src/Aether/Aether.fs

```diff
@@ -110,39 +110,39 @@ module Compose =
 
 (* Lenses
 
-   Various lenses implemented for common types such as tuples,
-   lists and maps, along with an id lens (which is useful for composing
-   a lens which has not specific "lensing" elements but is implicitly a chain
-   of one or more isomorphisms. Having an id lens enables the root composition. *)
+   Various lenses for common types such as tuples,
+   lists, and maps; along with an id_ lens (which is useful for composing
```

Comment:
I'm not sure that id_ is the right word here - it is an "id" lens (in the sense of being the identity function as a lens) - the underscore is stylistic/convention.